### ISSUE TITLE
Can use GXT ?

### ISSUE THREAD

**Issue by zzj1213** [nielsbaloe/vertxui#4]
Sencha's GXT is based on GWT, but with professional widgets.

**Comment by nielsbaloe** [comment]
Yes, GXT is cool. If you don't mind quite heavy-wight and commercial (1195 dollar/year) solutions, it's really awesome and usefull. I worked for more than 4 years with ext-js and we used the same graphical bits and pieces, it looked cool.

But, because it is so heavily dependend on GWT, you can not mix it with VertxUI. VertxUI only uses a little little bit GWT, only Elemental (no com.google.gwt..* classes!). GXT leans on GWT and as far as I can see it also builds on the regular GWT UI, the UiBinder and that .xml format,  the multiple compilation steps which generates different sets of .js files for each browser type (there is only 1 compilation with VertxUI/elemental), and the extra xml configuration file, etc. etc.

And you don't need it. Using any CSS framework you can achieve the same stuff. Well, perhaps all except the beautiful graphs (but there are enough other free libraries that can easily do this). VertxUI is more lightweight then GXT.

Your need for a component library is noted, and I'll create Bootstrap and PureCSS components (although these will be mostly one-liner classes!) for the next set of examples to come.

So in other words, no you can't, and unfortunately this will never happen unless someone pays me well to include it into VertxUI. Thanks for asking!

**Comment by remarkNS** [comment]
H(o)i @nielsbaloe, I completely agree with your comments about the GXT. Actually I think it was really cool some 2+ years ago, but I don't think it's cool any more. The same counts for smartgwt or alike libraries.

Bootstrap examples would be nice to see but what do you think about something like web components demo - e.g. polymer (2) paper components demo ?

More precisely I would really like to see how VertxUI would fit together with some complex web components like: https://github.com/vaadin/vaadin-grid (polymer web (free) component).

Any thoughts about this ?

Greetings, P.

**Comment by nielsbaloe** [comment]
Using vaadin-grid should be no problem at all. I'll write an example for that too.
